### DISCUSSION TITLE
Gallery image categories

### DISCUSSION THREAD

**Discussion by woodiertexas** [modrinth/.github#14] [Feature requests]
### Is your feature request related to a problem? Please describe.

Gallery images as of writing this are just scattered everywhere on the project's gallery page.

### Describe the solution you'd like

A feature allowing project owners to make gallery categories to sort the gallery images.

**Comment by Minenash** [comment]
Gallery sections sounds like a good idea

**Comment by Prospector** [comment]
Gallery images are now sortable
